fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
The system property names aws.accessKeyId and aws.secretKey were chosen specifically to coordinate with the documented AWS credentials chain, as described here: http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-roles.html. Changing them to add a fcrepo. prefix will work, but won't be consistent with an existing AWS pattern. The "aws.bucket" property was chosen simply to coordinate with the other two, so its name can be whatever is preferred.

With regard to security, using external repository.json files seems an appropriate recommendation to include in the documentation. Including these repository.json files in the baseline is useful for reference (this is what would be copied out to an external file) and use in development. I'm not excited about making these invalid json documents by adding comments in-line, given that modeshape could swap out their parser at any time.